cfme/fixtures/bzs.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Collection of fixtures for simplified work with bzs.
+
+The main purpose of this file is to add a pytest option for generates a BZ report, that
+gives information about the BZs that appear as coverage/automates metadata in the test_functions
+"""
+import pytest
+import yaml
+
+from cfme.fixtures.pytest_store import store
+from cfme.utils.blockers import BZ
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('Blocker options')
+    group.addoption(
+        '--generate-bz-report',
+        action='store_true',
+        default=False,
+        dest='generate_bz_report',
+        help='Generate a BZ report based on the automates/coverage metadata of test cases.'
+    )
+
+
+@pytest.mark.trylast
```

Comment:
**Question** Do we want to mark this with `trylast` here?

Reply:
Yeah this is leftover from some code I was looking at when writing this function. It is not necessary, so I'll remove it.